zoo/policies/cross-rl-agent/cross_rl_agent/train/utils.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+
+
+# ==============================================================================
+# -- Functions -----------------------------------------------------------------
+# ==============================================================================
```

Comment:
Perhaps not necessary to include this comment, to be inline with other `utils` files in the project?